conf/modules/allele_specific_calling.config

```diff
@@ -49,11 +49,30 @@ process {
         ]
     }
 
+    withName: '.*ALLELE_SPECIFIC_CALLING:SPLIT_MULTIALLELICS' {
+        ext.prefix = { "${meta.id}_norm" }
+        ext.args = '--output-type z --multiallelics -both'
+    }
+
     withName: '.*ALLELE_SPECIFIC_CALLING:TABIX_TABIX' {
         publishDir = [
             path: { "${params.outdir}/bootstrapann" },
             mode: params.publish_dir_mode,
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
         ]
     }
+
+    withName: '.*ALLELE_SPECIFIC_CALLING:REMOVE_DUPLICATES' {
+        ext.args = '--output-type z --rm-dup none'
+        ext.prefix = { "${meta.id}_split_rmdup" }
+    }
+
+    withName: '.*ALLELE_SPECIFIC_CALLING:ADD_VARCALLER_TO_BED' {
+        ext.args2   = '-s 1 -b 2 -e 3'
+    }
+
+    withName: '.*ALLELE_SPECIFIC_CALLING:BCFTOOLS_ANNOTATE' {
+        ext.args = "-c CHROM,FROM,TO,FOUND_IN --output-type z"
+        ext.prefix = { "${meta.id}_split_rmdup_info" }
+    }
```

Comment:
Let's publish this one as it is the final file from this subworkflow. We can also remove the publish from `BCFTOOLS_MERGE` and `RENAME_FILES`

Reply:
That's true :)

Reply:
done